packages/google-compute-engine-oslogin/nss_module/nss_oslogin.cc

```diff
@@ -101,6 +103,62 @@ int _nss_oslogin_getpwnam_r(const char *name, struct passwd *result,
   return NSS_STATUS_SUCCESS;
 }
 
+int _nss_oslogin_getgrby(string params, struct group *grp, char *buf, size_t buflen, int *errnop) {
```

Comment:
This method needs to handle pagination if metadata server returns next_page_token to retrieve all results.